examples/Peripherals/Registers/Reset_reason/Reset_reason.ino

```diff
@@ -36,18 +34,25 @@ static int default_button_state = LOW;
 void Reset_My_MCU() {
   // There are a few reset conditions. Keep the one you wish to use and comment out the others.
 
+  //Bellow is the WakeUp reset condition (needs STM32LowPower.h library)
+  //LowPower.shutdown(1000);
+
   // Below is the Software reset condition
-  // NVIC_SystemReset();
+  //NVIC_SystemReset();
 
   // Below is the Watchdog Timer reset condition
   IWatchdog.begin(1000);  //1ms tick then reset
   while (1)
     ;  // Wait for reset
+
+
 }
 
 void setup() {
   pinMode(USER_BTN_PIN, INPUT);
   default_button_state = digitalRead(USER_BTN_PIN);
+  //Serial.setRx(PA3);
+  //Serial.setTx(PA2);
```

Comment:
Don't think it i really helpful as it is not link to the subject and when user try this kind of example I assume they know how to change default Serial pins.
```suggestion
```